examples/gpu.yaml

```diff
@@ -0,0 +1,58 @@
+# Ray on Golem cluster name
+cluster_name: gpu-cluster
+
+# The maximum number of workers the cluster will have at any given time
+max_workers: 10
+
+# The cloud provider-specific configuration properties.
+provider:
+  type: "external"
+  use_internal_ips: true
+  module: "ray_on_golem.provider.node_provider.GolemNodeProvider"
+  parameters:
+    node_config:
+      # Parameters for golem demands (same for head and workers)
+      demand:
+        image_tag: "blueshade/raytest:0.9.0-py3.10-ray2.9.2-cuda11.8"
```

Comment:
Can we change `raytest`?

Reply:
I'm unsure if I get the question but anyway, yeah, maybe we can upload the image to the production repo in the registry when we release a new version... otherwise, we never put images from `develop` in the `golem/ray-on-golem` repo...

Reply:
`raytest` is a very raw name for an image. Can we change it to something more descriptive and accurate?

Reply:
that's not the name of the image, that's the name of the repository...  but anyway, yeah, we can